Made editorial updates (#488)
bigcommerce/dev-docs: docs/stencil-docs/developing-further/google-analytics-enhanced-ecommerce.md

</div>
</div>

### Downloading a Theme

> Data Tags will work on any theme. For this tutorial, we will be adding data tags to the Cornerstone theme. If you do not already have a local copy of Cornerstone on your machine, see [Downloading Cornerstone](/stencil-docs/installing-stencil-cli/installing-stencil#authorizing_download).

Reviewer: ```suggestion
Data tags will work on any theme. For this tutorial, we will be adding data tags to the Cornerstone theme. If you do not already have a local copy of Cornerstone on your machine, see [Downloading Cornerstone](/stencil-docs/installing-stencil-cli/installing-stencil#authorizing_download).

```

If you would like to implement Data Tags on your custom theme and do not already have a copy of your custom theme downloaded, see [Downloading a Marketplace Theme](/stencil-docs/installing-stencil-cli/installing-stencil#authorizing_download).

> **Note**: The remainder of this tutorial will be working off the theme’s base folder ‘cornerstone’.

Reviewer: ```suggestion
**Note**: The remainder of this tutorial will be working off the theme’s base folder `cornerstone`.

```

</div>
</div>

> ### Prerequisites

Reviewer: Prerequisites are being listed before mentioning that this is a tutorial. There needs to be some forewarning about this. The prerequisite section should come after the "Adding data tags" heading.

In addition, since this heading is going to remain h3, it should be removed from the "On this page" section.

* [BigCommerce Store](https://support.bigcommerce.com/s/article/Starting-a-Bigcommerce-Trial)
* [Optimized One-Page Checkout enabled](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout)

> ### Downloading a Theme

Reviewer: This section should be moved to within the "Add data tags" header.
Also, since we're using Cornerstone specifically in this tutorial, it should be added to the prerequisites.

Data Tags will work on any theme. For this tutorial, we will be adding data tags to the Cornerstone theme. If you do not already have a local copy of Cornerstone on your machine, see [Downloading Cornerstone](/stencil-docs/installing-stencil-cli/installing-stencil#authorizing_download).

If you would like to implement Data Tags on your custom theme and do not already have a copy of your custom theme downloaded, see [Downloading a Marketplace Theme](/stencil-docs/installing-stencil-cli/installing-stencil#authorizing_download).

> Note: The remainder of this tutorial will be working off the theme’s base folder ‘cornerstone’.

Reviewer: ```suggestion
**Note:** The remainder of this tutorial will be working off the theme’s base folder ‘cornerstone’.

```

## Prerequisites

> For this tutorial, you will need a BigCommerce store which uses Optimized One-Page Checkout for your store checkout.

Reviewer 2: Recommendations:
* Put prereqs in the introduction and change to an h3
* Split into two prereqs
* Keep it concise. 

Example:

```
### Prerequisites
* [BigCommerce Store](https://support.bigcommerce.com/s/article/Starting-a-Bigcommerce-Trial)
* [Optimized One-Page Checkout enabled](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout)
```

   <tr>
    <td><code>data-position</code></td>
       <td>The <code>data-position</code> tag is equal to the tracked product’s position or the tracked promotion’s position.</td>

>   	<td>Value is a string if creating the data tag for a promotion. The string should denote where the promotion is.

Reviewer: ```suggestion
  	<td>Value is a string if creating the data tag for a promotion. The string should denote the location of the promotion.

```
